app/layout.tsx

```diff
@@ -1,22 +1,33 @@
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
-import "./globals.css";
+import "./global.css";
+import { Button, Paper, buttonGroupClasses } from "@mui/material";
+import Image from "next/image";
 
-const inter = Inter({ subsets: ["latin"] });
-
-export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+export const metadata = {
+	title: "Game Trucker",
+	description: "Generated by Next.js",
```

Comment:
I can't see where it is being used, I'm assuming that's some kind of leftover code (?) 😃

Reply:
"Next.js has a Metadata API that can be used to define your application metadata (e.g. meta and link tags inside your HTML head element) for improved SEO and web shareability."